noxfile.py

```diff
@@ -49,6 +49,12 @@ def unit(session, proto="python"):
     session.install("-e", ".[testing]", "-c", constraints_path)
     if proto == "cpp":  # 4.20 does not have cpp.
         session.install("protobuf==3.19.0")
+
+    # The warnings as errors flag `-W=error` was removed in
```

Comment:
Worth filing an issue for this, and linking it here.

Reply:
Done in https://github.com/googleapis/proto-plus-python/issues/403

Reply:
Oh, I see. #403 is the issue tracking this. So fixing #403 MEANS re-enabling the warning.
```suggestion
    # TODO(https://github.com/googleapis/proto-plus-python/issues/403): re-enable -W=error
    # The warnings-as\-errors flag `-W=error` was removed in
    # https://github.com/googleapis/proto-plus-python/pull/400.
    # It should be re-added once issue https://github.com/protocolbuffers/protobuf/issues/12186 is fixed. 
```